### DISCUSSION TITLE
SigenStor - Backup reserve

### DISCUSSION THREAD

**Discussion by Ramon1347** [TypQxQ/Sigenergy-Home-Assistant-Integration#25] [General]
Hey, i wanted to know if there is a opportunity to set the backup reserve with this integration? And how is it possible ?
Best regards :)

**Comment by TypQxQ** [comment]
Hi! Welcome to the group.
What is the "backup reserve"?
REgards, Andrei

**Reply by Ramon1347** [reply]
Hi, 
the backup reserve is the battery capacity that is retained in the event of a power failure. I want to control this in order to automatically increase the backup reserve if a strong storm is forecast, for example. This will also automatically charge the storage at maximum power to this value if the storage SOC is below this value. 
Can you please show us how this feature can be add to your integration :) ?
![IMG_1247](https://github.com/user-attachments/assets/eb45e5dd-4c13-4d0e-a0a7-f4cabe7c9b5a)
![IMG_1248](https://github.com/user-attachments/assets/ec07bc22-a1f2-4d07-bb96-dfb94c11de50)

**Reply by EnterTheMaelstrom** [reply]
I do not have a gateway, so I don't have the option in the app... maybe this would be register 40032 / 40034 / 40036 ?
Do those change accordingly when you change the values in the app ?

**Comment by TypQxQ** [comment]
Register 40032 / 40034 / 40036 has nothing to do with that unfortunatley.
I haven't found a setting for this.
Regards, Andrei

**Reply by Ramon1347** [reply]
Ah ok, thank you for checking. How can i read all the information and all the register which are send from the sigenstor to find the backup reserve information? Or do you allready checked every sended information from the sigenstor ?

**Reply by micvppl** [reply]
I believe it is only doable with HA automations and cofigs as those are sigen app software options to decide when and what to do with components. 
This integration is not copy of sigen app but only modbus integration that give You set of tools and You need to decide what to do with them :)
